zanata-war/src/main/java/org/zanata/action/MergeTransAction.java

```diff
@@ -0,0 +1,230 @@
+package org.zanata.action;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.List;
+
+import javax.faces.application.FacesMessage;
+import javax.faces.component.UIOutput;
+import javax.faces.event.AjaxBehaviorEvent;
+import javax.faces.event.ValueChangeEvent;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import org.apache.commons.lang.StringUtils;
+import org.jboss.seam.ScopeType;
+import org.jboss.seam.annotations.Create;
+import org.jboss.seam.annotations.In;
+import org.jboss.seam.annotations.Name;
+import org.jboss.seam.annotations.Scope;
+import org.jboss.seam.faces.FacesMessages;
+import org.jboss.seam.international.StatusMessage;
+import org.zanata.async.handle.MergeTranslationsTaskHandle;
+import org.zanata.common.EntityStatus;
+import org.zanata.dao.ProjectDAO;
+import org.zanata.dao.ProjectIterationDAO;
+import org.zanata.i18n.Messages;
+import org.zanata.model.HProject;
+import org.zanata.model.HProjectIteration;
+import org.zanata.ui.CopyAction;
+
+/**
+ * Handles action from merge_trans_modal.xhtml
```

Comment:
It is helpful to mention the template file that calls this class's methods. On its own, it does not give developers a quick overview of what this is about though (reading through xhtml to see what it does is not quick or easy), so can you write a sentence or two about the responsibilities this class has?

I imagine it would be something about handling actions from that template to start translation merges between different project-versions, and providing methods to display their progress.